src/include/daos_pool.h

```diff
@@ -162,8 +162,8 @@ enum daos_pool_info_bit {
 	DPI_ENGINES_ENABLED = 1ULL << 2,
 	/** true to include (in \a ranks) engines with some or all targets disabled (down). */
 	DPI_ENGINES_DISABLED = 1ULL << 3,
-	/** true to include (in \a ranks) suspect engines. */
-	DPI_ENGINES_SUSPECT = 1ULL << 4,
+	/** true to include (in \a ranks) engines marked DEAD by SWIM. */
```

Comment:
[Nit; not requesting changes] Maybe this degree of inaccuracy is acceptable, but I just want to clarify that these "dead" engines are actually the set of pool engines who are not "alive". In other words, they may also include engines absent from the SWIM membership.

Reply:
if that is the case then maybe suspect is a more accurate term?

Reply:
@liw: When would we have engines in the pool map that are absent from the SWIM membership, but aren't dead?

Reply:
When the MS has excluded the engines from the system (and the new system map has been distributed out), but the PS hasn't excluded the engines from the pool. In this state, from the pool's point of view, these engines are not strictly SWIM-DEAD, just unavailable---loosely speaking, "dead".

Hmm, personally I don't think "suspect" is more accurate. :)